src/lib/api.js

```diff
@@ -37,6 +60,13 @@ export default function(opts) {
         return item;
     }
 
+    async function findByURL(url) {
+        // escaping regex (http://stackoverflow.com/a/3561711)
+        url = url.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
+        const regex = new RegExp(`.*${url}.*`, 'i');
+        return RequestModel.findAsync({ 'url': regex });
+    }
```

Comment:
I just figured since its two small changed, would be quicker to review/release at once. 

I don't mind removing this and putting it a different PR.